.changeset/seven-penguins-fly.md

```diff
@@ -0,0 +1,30 @@
+---
+"@salt-ds/lab": minor
+"@salt-ds/theme": minor
```

Comment:
Minor but can we split the theme and labs changelog please, that way each change shows against the specific library.

Reply:
Done!